jobs/ansible-deploy-aws/ansible-deploy-aws.groovy

```diff
@@ -48,23 +48,17 @@ virtualenv .venv
 [[ -f /usr/bin/figlet ]] && figlet Installing python dependencies
 pip install -Ur requirements.txt
 
-
 [[ -f /usr/bin/figlet ]] \
   && figlet Refreshing ec2 inventory cache
 ./ec2.py --refresh-cache > /dev/null 2>&1 && echo done...
 
 [[ -f /usr/bin/figlet ]] \
   && figlet Configure ansible to use ${DEPLOY_ENV} environment
-make check-env-var render-ssh-config clean-roles ansible-galaxy 
+make clean-roles ansible-galaxy 
```

Comment:
Should we take the opportunity to make these a pre-req of running the `aws` or `gce` targets?